src/shared/components/status-icon-assessment/status-icon-assessment.tsx

```diff
@@ -0,0 +1,95 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+
+import { Flex, FlexItem, SpinnerProps } from "@patternfly/react-core";
+import {
+  CheckCircleIcon,
+  ErrorCircleOIcon,
+  InProgressIcon,
+} from "@patternfly/react-icons";
+import { SVGIconProps } from "@patternfly/react-icons/dist/js/createIcon";
+import {
+  global_disabled_color_200 as disabledColor,
+  global_success_color_100 as successColor,
+  global_Color_dark_200 as unknownColor,
+  global_info_color_200 as loadingColor,
+} from "@patternfly/react-tokens";
+
+export type StatusIconAssessmentType =
+  | "NotStarted"
+  | "InProgress"
+  | "Completed";
+
+type IconListType = {
+  [key in StatusIconAssessmentType]: {
+    Icon:
+      | React.ComponentClass<SVGIconProps>
+      | React.FunctionComponent<SpinnerProps>;
+    color: { name: string; value: string; var: string };
+  };
+};
+const iconList: IconListType = {
+  NotStarted: {
+    Icon: ErrorCircleOIcon,
```

Comment:
From mockups it seems like the icon should be dark with a white `x` 
![Screenshot from 2021-04-07 13-08-45](https://user-images.githubusercontent.com/7288588/113857123-6a5e6780-97a2-11eb-9059-a7a307a7220a.png)
but it seems like the opposite has been used
![Screenshot from 2021-04-07 13-09-33](https://user-images.githubusercontent.com/7288588/113857209-84984580-97a2-11eb-8dd7-13f07aa678a0.png)

Maybe `TimesCircleIcon` icons can be the right one?

Reply:
Good point, my last commit changed the icon. Thanks @mrizzi